<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1377</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenEDX_converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="Input_and_console">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Введите путь к исходному файлу (.docx):</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="lineEdit_file"/>
        </item>
        <item>
         <widget class="QPushButton" name="file_browse">
          <property name="text">
           <string>Обзор</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Введите путь к каталогу, в который будет помещён результат:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="lineEdit_dir">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dir_browse">
          <property name="text">
           <string>Обзор</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <widget class="QPushButton" name="convert">
        <property name="text">
         <string>Конвертировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="Options">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="title">
       <string>Настройки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>829</width>
            <height>845</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="Library_options">
             <property name="title">
              <string>Библиотека</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_library">
                  <property name="text">
                   <string>Имя библиотеки</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit_library">
                  <property name="inputMethodHints">
                   <set>Qt::ImhUrlCharactersOnly</set>
                  </property>
                  <property name="text">
                   <string>Test_Library</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_display_name">
                  <property name="text">
                   <string>Выводимое имя</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEdit_display_name">
                  <property name="text">
                   <string>Тест</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_org">
                  <property name="text">
                   <string>Название организации</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEdit_org">
                  <property name="inputMethodHints">
                   <set>Qt::ImhNone</set>
                  </property>
                  <property name="text">
                   <string>LETI</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_use_latex_compiler">
                  <property name="text">
                   <string>Использовать компилятор latex</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QCheckBox" name="checkBox_use_latex_compiler">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_show_correctness">
                  <property name="text">
                   <string>Показывать корректность</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QComboBox" name="comboBox_show_correctness">
                  <property name="editable">
                   <bool>false</bool>
                  </property>
                  <property name="currentText">
                   <string>&lt;не добавлять&gt;</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>&lt;не добавлять&gt;</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Всегда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Никогда</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_showanswer">
                  <property name="text">
                   <string>Показывать ответ</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QComboBox" name="comboBox_showanswer">
                  <item>
                   <property name="text">
                    <string>&lt;не добавлять&gt;</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Выполнен</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Всегда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Дан ответ</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Использована попытка</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Закрыт</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ответ верен или прошёл срок сдачи</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Прошёл срок сдачи</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Никогда</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_rerandomize">
                  <property name="text">
                   <string>Выбор случайных значений</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QComboBox" name="comboBox_rerandomize">
                  <item>
                   <property name="text">
                    <string>Никогда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Всегда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>При сбросе</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Для каждого студента</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_show_reset_button">
                  <property name="text">
                   <string>Показывать кнопку &quot;Cбросить&quot;</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QCheckBox" name="checkBox_show_reset_button">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="9" column="0">
                 <widget class="QLabel" name="label_max_attempts">
                  <property name="text">
                   <string>Максимальное количество попыток</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="1">
                 <widget class="QSpinBox" name="spinBox_max_attempts"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_time">
                  <property name="text">
                   <string>Добавлять временную метку</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="checkBox_time">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Вопросы</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QFormLayout" name="formLayout_2">
                <item row="3" column="0">
                 <widget class="QLabel" name="label_show_correctness_q">
                  <property name="text">
                   <string>Показывать корректность</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="comboBox_show_correctness_q">
                  <item>
                   <property name="text">
                    <string>Всегда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Никогда</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_showanswer_q">
                  <property name="text">
                   <string>Показывать ответ</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QComboBox" name="comboBox_showanswer_q">
                  <item>
                   <property name="text">
                    <string>Никогда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Всегда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Дан ответ</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Использована попытка</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Закрыт</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Выполнен</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ответ верен или прошёл срок сдачи</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Прошёл срок сдачи</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_rerandomize_q">
                  <property name="text">
                   <string>Выбор случайных значений</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QComboBox" name="comboBox_rerandomize_q">
                  <item>
                   <property name="text">
                    <string>Для каждого студента</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Всегда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>При сбросе</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Никогда</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_max_attempts_q">
                  <property name="text">
                   <string>Максимальное количество попыток</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QSpinBox" name="spinBox_max_attempts_q">
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_weight">
                  <property name="text">
                   <string>Вес</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_weight_q">
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_show_reset_button_q">
                  <property name="text">
                   <string>Показывать кнопку &quot;Cбросить&quot;</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QCheckBox" name="checkBox_show_reset_button_q">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="9" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Погрешность (где применимо)</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="1">
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="text">
                   <string>1%</string>
                  </property>
                 </widget>
                </item>
                <item row="15" column="0">
                 <widget class="QLabel" name="label_submission_wait_seconds_q">
                  <property name="text">
                   <string>Время между попытками</string>
                  </property>
                 </widget>
                </item>
                <item row="15" column="1">
                 <widget class="QSpinBox" name="spinBox_submission_wait_seconds_q"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_individually">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Изменять настройки для каждого вопроса отдельно</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::PlainText</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_display_name_q">
                  <property name="text">
                   <string>Выводимое имя</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEdit_display_name_q">
                  <property name="text">
                   <string>Вопрос</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="checkBox_individually">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="10" column="0">
                 <widget class="QLabel" name="label_reg_type_t">
                  <property name="text">
                   <string>Тип полей для ввода текста</string>
                  </property>
                 </widget>
                </item>
                <item row="11" column="0">
                 <widget class="QLabel" name="label_reg_type_m">
                  <property name="text">
                   <string>Тип полей для ввода мат. выражений</string>
                  </property>
                 </widget>
                </item>
                <item row="13" column="0">
                 <widget class="QLabel" name="label_trailing_text_m">
                  <property name="text">
                   <string>Текст после поля ввода математичесого выражения</string>
                  </property>
                 </widget>
                </item>
                <item row="14" column="0">
                 <widget class="QLabel" name="label_trailing_text_n">
                  <property name="text">
                   <string>Текст после поля ввода численного ответа</string>
                  </property>
                 </widget>
                </item>
                <item row="10" column="1">
                 <widget class="QComboBox" name="comboBox_reg_type_t">
                  <item>
                   <property name="text">
                    <string>Чувств. к регистру</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Нечувств. к регистру</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Для рег. выражений, чувств. к регистру</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Для рег. выражений, нечувств. к регистру</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="11" column="1">
                 <widget class="QComboBox" name="comboBox_reg_type_m">
                  <item>
                   <property name="text">
                    <string>Чувств. к регистру</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Нечувств. к регистру</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="13" column="1">
                 <widget class="QLineEdit" name="lineEdit_trailing_text_m"/>
                </item>
                <item row="14" column="1">
                 <widget class="QLineEdit" name="lineEdit_trailing_text_n"/>
                </item>
                <item row="12" column="0">
                 <widget class="QLabel" name="label_trailing_text_t">
                  <property name="text">
                   <string>Текст после поля ввода текста</string>
                  </property>
                 </widget>
                </item>
                <item row="12" column="1">
                 <widget class="QLineEdit" name="lineEdit_trailing_text_t"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
